### DISCUSSION TITLE
User activation process optimization

### DISCUSSION THREAD

**Discussion by svanschu** [joomla/joomla-cms#47165] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
We have a club page with an internal area. The Membership is checked manually with a list of names and club numbers. The process looks like:

1. Administrator get the mail of a user registration
2. Admin logs into administrator panel
3. Admin goes to users
4. opens the specific account
5. checks the member number and name
6. set the ACL group to a predefined group/level (member in this example)
7. Save and Close the user profile
8. admin log into frontend
9. back to e-mail
10. clicking on the link to activate and notify the new member user account

This process is really time intense.
You can activate the user account in backend, but then the user does not get a notification.

An additional problem is, that sometimes those registration mails don't appear in my inbox (2 out of 10). In this case you don't have the chance to get the same functionality as with the link in the e-mail.

### Describe the solution you'd like
A possible solution would be a possibility (new button?) to get in backend the same functionality as the link in the registration mail has. It would make the process much faster and more efficient.

**Comment by brianteeman** [comment]
Is this what you are looking for #20282

**Comment by svanschu** [comment]
Hi, @brianteeman 
yes, this PR has the same topic. Do I understand all of those comments right, that it will be a J4.0 feature?
